tests/phpunit/tests/post/revisions.php

```diff
@@ -655,6 +655,54 @@ public function test_wp_save_post_revision_error() {
 		$this->assertWPError( $revision );
 	}
 
+	/**
+	 * Tests that wp_get_last_revision_id_and_total_count() returns last revision id and total count.
+	 *
+	 * @ticket 55857
+	 * @dataProvider data_wp_get_post_revisions_url
+	 */
+	public function test_wp_get_last_revision_id_and_total_count( $revisions ) {
+		$post_id = self::factory()->post->create();
+
+		for ( $i = 0; $i < $revisions; ++$i ) {
+			wp_update_post(
+				array(
+					'ID'         => $post_id,
+					'post_title' => 'Some Post ' . $i,
+				)
+			);
+
+			$post_revisions     = wp_get_post_revisions( $post_id );
+			$last_post_revision = current( $post_revisions );
+			$revision           = wp_get_last_revision_id_and_total_count( $post_id );
+
+			$this->assertSame(
+				$last_post_revision->ID,
+				$revision['revision'],
+				'Failed asserting latest revision id.' . print_r( $post_revisions, true )
```

Comment:
I think these messages are just designed to be text and not have dynamic content in them.